src/install/python/python-options/python-agent-unsupported-frameworks.mdx

```diff
@@ -0,0 +1,90 @@
+---
+componentType: appInfo
+optionType: framework
+headingText: Review the manual initialization steps for unsupported frameworks
+---
+
+It sounds like you're using an unsupported framework, such as mod_wsgi or WebFaction. Here's what you need to know:
+
+### Manual initialization in app code [#manual-integration]
```

Comment:
@TimPansino Do you think we need to make another page for manual integration of ASGI or do you think we could just modify this existing WSGI manual setup wording to be more generic to work for both WSGI and ASGI? I'm not as familiar with ASGI as you are and I wasn't sure how much different the setup would be.

Reply:
I would like a separate section for WSGI and ASGI calling out the 2 application wrapper APIs. That seems like it would be the easiest for customers to understand.